jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlParserTest.java

```diff
@@ -31,12 +32,21 @@ public class XmlParserTest
     @Test
     public void testXmlParser() throws Exception
     {
-        XmlParser parser = new XmlParser();
+        XmlParser parser = new XmlParser()
+        {
+            @Override
+            protected InputSource resolveEntity(String pid, String sid)
+            {
+                InputSource inputSource = super.resolveEntity(pid, sid);
+                assertNotNull(inputSource, "You are using entities in your XML that don't match your redirectEntity mappings: pid=" + pid + ", sid=" + sid);
```

Comment:
This ensures that the testcase remains sane.

Looks like the XML was updated without realizing that this testcase exists.
The XML no longer referred to entities that were being redirected internally.